astropy/io/ascii/core.py

```diff
@@ -1463,7 +1470,8 @@ def _get_writer(Writer, fast_writer, **kwargs):
     if 'exclude_names' in kwargs:
         writer.exclude_names = kwargs['exclude_names']
     if 'fill_values' in kwargs:
-        writer.data.fill_values = kwargs['fill_values']
+        # Prepend user-specified values to the class default.  Both are lists.
+        writer.data.fill_values = kwargs['fill_values'] + writer.data.fill_values
```

Comment:
This breaks some tests because one is a tuple and the other a list. Not sure why (user/test-input?) but maybe this needs a `tuple` or `list` call around it (or `itertools.chain` and a cast afterwards).